llvm/lib/TableGen/Error.cpp

```diff
@@ -170,4 +170,11 @@ void CheckAssert(SMLoc Loc, Init *Condition, Init *Message) {
   }
 }
 
+void CheckDump(SMLoc Loc, Init *Message) {
```

Comment:
This is not a `check` I think, it does the `dump`.
Change it to `Dump`, `DoDump` or other more precise name?

Reply:
Yep - I've gone with `dumpMessage`